challenge.js

```diff
@@ -6,10 +6,10 @@
  *		to a web page without refreshing it?
  *
  *	Answer below.
- */
+ */ //create a new element first, and then append it to an existing element.
 
 // ANSWER
-
+//create a new element first, and then append it to an existing element.
```

Comment:
Yes, this is one way to do it, but your answer is not specific enough. 1/2 credit.